source/NVDAObjects/IAccessible/sysListView32.py

```diff
@@ -276,7 +276,7 @@ def _get_value(self):
 			value=self.displayText
 		if not value:
 			return None
-		#Some list view items in Vista can contain annoying left-to-right and right-to-left indicator characters which really should not be there.
+		#Some list view items in Vista and later can contain annoying left-to-right and right-to-left indicator characters which really should not be there.

```

Comment:
Vista support is no more, so you could consider changing this to 7...

Reply:
For historical tracking, I'd rather keep it this way. It is good to know when this behavior started.